index.html

```diff
@@ -954,7 +954,7 @@ <h2>DID Parameters</h2>
 are not supported by all <a>DID methods</a>. Where optional parameters are
 supported, they are expected to operate uniformly across the <a>DID methods</a>
 that do support them. The following table provides common DID parameters
-that function the same way across all <a>DID methods</a>.
+that function the same way across all <a>DID methods</a>. For all parameters the support is OPTIONAL.
```

Comment:
```suggestion
that function the same way across all <a>DID methods</a>. Support of all 
parameters listed in the table below is OPTIONAL.
```

Reply:
```suggestion
that function the same way across all <a>DID methods</a>. Support for all <a>DID Parameters</a> is OPTIONAL.
```

It's not just for everything in the table below... at this point, we have no mandatory DID Parameters, which is fine... but we should also make it clear that people adding stuff to the DID Registry can't make their additions MANDATORY. The wording above does that w/o going to great lengths to explain it ('cause the liklihood of someone doing that is quite low).

Reply:
fair enough